src/variant.cpp

```diff
@@ -552,6 +552,22 @@ namespace {
         v->capturesToHand = true;
         return v;
     }
+    // Dragon Chess
+    // 8x8 variant invented by Miguel Illescas:
+    // https://www.edami.com/dragonchess/
+    // Like regular chess, but with an extra piece, the dragon, which moves like
+    // an archbishop (i.e. bishop+knight). The dragon can be dropped at an empty
+    // square on the back rank instead of making a normal move.
+    Variant* dragon_variant() {
+        Variant *v = chess_variant_base()->init();
+        v->variantTemplate = "dragon";
```

Comment:
The variantTemplate is only used in communication with Winboard/Xboard (or more generally in the CECP protocol) and refers to built-in variants of the GUI to inherit logic from: https://github.com/ianfab/Fairy-Stockfish/blob/67209f150d44b6ac5fcc01659442fcc9a4815e95/src/variants.ini#L137
Therefore this should equal to one of the predefined template variants in XBoard/Winboard. For most chess based variants `fairy` usually is the recommended choice, but for variants with drops where captured pieces don't return to the opponent's hand (like in crazyhouse/shogi), usually `bughouse` should be used.